src/periodic_report_population.rs

```diff
@@ -0,0 +1,89 @@
+use crate::population_loader::{Age, CensusTract};
+
+use crate::Parameters;
+use ixa::{
+    context::{Context, ExecutionPhase},
+    create_report_trait, define_data_plugin,
+    error::IxaError,
+    global_properties::ContextGlobalPropertiesExt,
+    people::{ContextPeopleExt, PersonCreatedEvent},
+    report::{ContextReportExt, Report},
+};
+use serde::{Deserialize, Serialize};
+use std::path::{Path, PathBuf};
+
+use std::collections::HashSet;
+
+#[derive(Serialize, Deserialize, Clone, PartialEq)]
+struct PopulationReportItem {
+    time: f64,
+    census_tract: usize,
+    age: u8,
+    population: usize,
+}
+
+#[derive(Clone)]
+struct PopulationReportData {
+    census_tract_set: HashSet<usize>,
+}
+
+define_data_plugin!(
+    PopulationReportPlugin,
+    PopulationReportData,
+    PopulationReportData {
+        census_tract_set: HashSet::new(),
+    }
+);
+
+create_report_trait!(PopulationReportItem);
+
+fn send_population_report(context: &mut Context) {
+    let population_data = context.get_data_container(PopulationReportPlugin).unwrap();
+
+    let current_census_set = &population_data.census_tract_set;
+    for age_it in 0..100 {
```

Comment:
What does `age_it` stand for?

Reply:
It stands for age_iterator but this is a bad name, maybe. I can change that.